test/lib/assert-equal-dom.js

```diff
@@ -31,7 +32,12 @@ function areEqual(a, b) {
                 return equalStyle(a[key], b[key])
             }
             if (typeof a === "object" || typeof a === "function") {
-                if (!areEqual(a[key], b[key])) {
+                if (cache.indexOf(a) !== -1) {
```

Comment:
i forget which is the lowest version of IE @Matt-Esch wants to support but IE9 is the earliest to support `Array.prototype.indexOf` and i think the plan is to support earlier than that.